atd-vze/src/views/Tools/CreateCrashRecordTable.js

```diff
@@ -147,15 +135,15 @@ const CreateCrashRecordTable = ({ crashesData, loading, error }) => {
                           </Link>
                         </td>
                         <td>{item.case_id}</td>
-                        <td>{item.crash_timestamp}</td>
+                        <td>{formatDateTimeString(item.crash_timestamp)}</td>
```

Comment:
previously these werent formatted and were showing up as ugly ISO strings

Reply:
nice!